src/openvic-simulation/research/Invention.cpp

```diff
@@ -74,15 +79,19 @@ bool InventionManager::load_inventions_file(
 			ConditionScript limit { scope_t::COUNTRY, scope_t::COUNTRY, scope_t::NO_SCOPE };
 			ConditionalWeight chance { scope_t::COUNTRY, scope_t::COUNTRY, scope_t::NO_SCOPE };
 
-			bool ret = modifier_manager.expect_modifier_value_and_keys(move_variable_callback(loose_modifiers),
+			bool ret = modifier_manager.expect_modifier_value_and_keys(
+				move_variable_callback(loose_modifiers),
+				INVENTION,
```

Comment:
The modifier type enum is used for modifier variants in some cases, but it's also used for UI logic (e.g. some tooltips group all tech contributions into a single "From tech" line). Although the `INVENTION` argument doesn't do anything special here, we need something to fill the argument slot (and to let the function know this isn't a modifier using effect variants) so might as well use the specific type rather than a less descriptive generic null value.